lib/block_view_validator.go

```diff
@@ -292,7 +292,6 @@ type RegisterAsValidatorTxindexMetadata struct {
 	VotingPublicKey               string
 	VotingPublicKeySignature      string
 	VotingSignatureBlockHeight    uint64
-	UnstakedStakers               []*UnstakedStakerTxindexMetadata
```

Comment:
We no longer will have unstaked stakers when registering as a validator. Instead we validate that a user can't update DisableDelegatedStake=true if they have existing delegated stakers.